evap/grades/views.py

```diff
@@ -117,29 +117,19 @@ def upload_grades(request, semester_id, course_id):
 
 
 @grade_publisher_required
-def toggle_no_grades(request, semester_id, course_id):
-    semester = get_object_or_404(Semester, id=semester_id)
+def toggle_no_grades(request):
+    course_id = request.POST.get("course_id")
     course = get_object_or_404(Course, id=course_id)
```

Comment:
well whether the semester should be part of it is another discussion.

what i meant is continue using the URL to contain the variable(s), just like we do for GET requests. in other words, why change this, what's the advantage?